cdc/owner/changefeed.go

```diff
@@ -296,11 +303,7 @@ func (c *changefeed) tick(ctx cdcContext.Context, captures map[model.CaptureID]*
 		return errors.Trace(err)
 	default:
 	}
-	// we need to wait ddl ddlSink to be ready before we do the other things
-	// otherwise, we may cause a nil pointer panic when we try to write to the ddl ddlSink.
-	if !c.ddlSink.isInitialized() {
-		return nil
-	}
+
```

Comment:
why remove ddl sink initialization check here

Reply:
The DDL internal sink is initialized lazily now, and it's not required that the sink must be initialized when interfaces are called.